desktop/core/src/desktop/lib/raz/raz_client_test.py

```diff
@@ -118,18 +118,53 @@ def test_get_raz_client(self):
         assert_true(isinstance(client, RazClient))
 
         HTTPKerberosAuth.assert_called()
-        assert_equal(
-          client.raz_url, self.raz_url
-        )
-        assert_equal(
-          client.service_name, 'gethue_s3'
-        )
-        assert_equal(
-          client.cluster_name, 'gethueCluster'
+        assert_equal(client.raz_url, self.raz_url)
+        assert_equal(client.service_name, 'gethue_s3')
+        assert_equal(client.cluster_name, 'gethueCluster')
+
+
+  def test_check_access_adls(self):
+      raz_token = Mock()
+
+      client = RazClient(
+        self.raz_url, 
+        raz_token, 
+        username=self.username, 
+        service="adls", 
+        service_name="adls", 
+        cluster_name="cl1",
+        operation={
+          "resource": {
+            "storageaccount": "test",
+            "container": "test",
+            "relativepath": "data"
+          },
+          "resourceOwner": "",
+          "action": "read",
+          "accessTypes":["read"]
```

Comment:
read or write should be auto generated depending if POST/GET/UPDATE... request

Reply:
@romainr  IIRC, isn't this for S3 and not ADLS? Or are we following the mapping for ADLS also?

In CDPD-25742, Read access is there for a POST curl request

Reply:
To dig more for the logic so. Better to ask them if we can deduct it from the ranger code or will have to do our own mapping